mockgcp/mockasset/savedquery.go

```diff
@@ -70,6 +70,26 @@ func (s *AssetService) CreateSavedQuery(ctx context.Context, req *pb.CreateSaved
 
 	now := time.Now()
 
+	if req.SavedQuery.GetContent() != nil &&
```

Comment:
You can replace all that with `if t := req.SavedQuery.GetContent().GetIamPolicyAnalysisQuery().GetConditionContext().GetAccessTime(); t !=nil {`

(Possibly even the AsTime() call also, but I'm less sure about that one!)

Reply:
Done.